src/Phinx/Config/Config.php

```diff
@@ -49,6 +49,11 @@ class Config implements ConfigInterface
     protected $configFilePath;
 
     /**
+     * @var $phinx_version
+     */
+    public static $phinx_version = "0.4.1";
```

Comment:
Okay, I'll remove this. To me it seems messy to have to define PhinxApplication with a version number though, for what seems like purely to be used for the initial output line to report the version. That number can't be trusted much can it?